puppet-mode.el

```diff
@@ -1109,6 +1117,69 @@ for each entry."
 ;;;###autoload
 (add-to-list 'auto-mode-alist '("\\.pp\\'" . puppet-mode))
 
+;; this are only the mostly used puppet code keywords
+;; ( sorted by the order they are used in a puppet class )
+(setq puppet-kwdList
```

Comment:
Naming conventions: No abbreviations, and `snake-case`.  IOW, it's `puppet-keywords` here.